src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -271,29 +265,27 @@ function compute_diffusive_fluxes(
         ),
     )
 
-    @. aux_tc_f.diffusive_flux_qt =
-        -aux_tc_f.ρ_ae_KH * ∇q_tot_en(wvec(aux_en.q_tot))
-    @. aux_tc_f.diffusive_flux_h =
-        -aux_tc_f.ρ_ae_KH * ∇h_tot_en(wvec(aux_en.h_tot))
+    @. aux_tc_f.diffusive_flux_qt = -aux_tc_f.ρ_ae_KH * ∇q_tot_en(aux_en.q_tot)
+    @. aux_tc_f.diffusive_flux_h = -aux_tc_f.ρ_ae_KH * ∇h_tot_en(aux_en.h_tot)
     @. aux_tc_f.diffusive_flux_uₕ = -aux_tc_f.ρ_ae_KM * ∇uₕ_gm(prog_gm_uₕ)
 
     if edmf.moisture_model isa NonEquilMoistModel
         aeKHq_liq_bc = FT(0)
         aeKHq_ice_bc = FT(0)
 
-        ∇q_liq_en = CCO.DivergenceC2F(;
-            bottom = CCO.SetDivergence(aeKHq_liq_bc),
-            top = CCO.SetDivergence(FT(0)),
+        ∇q_liq_en = CCO.GradientC2F(;
+            bottom = CCO.SetGradient(CCG.WVector(aeKHq_liq_bc)),
+            top = CCO.SetGradient(CCG.Covariant3Vector(FT(0))),
         )
```

Comment:
Why do we want our bottom condition to be based on `CCG.WVector` but the top to be based on `CCG.Covariant3Vector` ?